test/e2e/common/utils.py

```diff
@@ -244,3 +238,13 @@ def predict_modelmesh(service_name, input_json, pod_name, model_name=None):
             url = f"http://localhost:8008/v2/models/{model_name}/infer"
             response = requests.post(url, json.dumps(data))
             return json.loads(response.content.decode("utf-8"))
+
+
+def get_isvc_endpoint(isvc):
+    host = urlparse(isvc["status"]["url"]).netloc
+    path = urlparse(isvc["status"]["url"]).path
+    if os.environ.get("CI_USE_ISVC_HOST") == "1":
+        cluster_ip = host
+    else:
+        cluster_ip = get_cluster_ip()
+    return cluster_ip, host, path
```

Comment:
new line

Reply:
New line before `return`?

Reply:
at the end of the file :)

Reply:
Fixed.

Reply:
Sorry, I reverted.
The python linter didn't like it. See: https://github.com/opendatahub-io/kserve/actions/runs/7269665250/job/19807632226#step:4:38

Reply:
hmm, not sure why.
PEP8 suggests a line terminator while this custom linter asks to remove.